.gitignore

```diff
@@ -2,3 +2,5 @@
 /bin/
 .direnv
 .envrc
+flake.lock
```

Comment:
There's no point in both adding a file and ignoring it, ignores only work for untracked files.